fig/includes.py

```diff
@@ -0,0 +1,190 @@
+"""Include external projects, allowing services to link to a service
+defined in an external project.
+"""
+import logging
+import os
+import time
+
+from pytimeparse import timeparse
+import requests
+import requests.exceptions
+import six
+from six.moves.urllib.parse import urlparse, quote
+import yaml
+
+from fig.service import ConfigError
+
+
+log = logging.getLogger(__name__)
+
+
+class FetchExternalConfigError(Exception):
+    pass
+
+
+def normalize_url(url):
+    url = urlparse(url)
+    return url if url.scheme else url._replace(scheme='file')
+
+
+def read_config(content):
+    return yaml.safe_load(content)
+
+
+def get_project_from_file(url):
+    # Handle urls in the form file://./some/relative/path
```

Comment:
Does this handle files in local directories?  If not, it would be great if it did :)